pkg/collector/metric/utils.go

```diff
@@ -184,6 +187,54 @@ func getS390xArchitecture() (string, error) {
 	return fmt.Sprintf("zSystems model %s", strings.TrimSpace(uarch[1])), err
 }
 
+// getCPUModel() provides alternative CPU architecture detection mechanism other than aditional tools dependencies, such as 'cpuid', 'archspec', etc.
+// Take Intel Xeon 4th Gen Scalable Processor(Codename: Sapphire Rapids) as example, in normalized_cpu_arch.csv:
+// Vendor,Processor,Model,Microarchitecture
+// ...
+// Intel,SAPPHIRERAPIDS_X,143,Golden Cove
+// ...
+//
+// We can use the 'lscpu' command, which exists in most of the Linux Distros, to fetch the CPU microarchitecture information like this:
+// $ lscpu |grep Model:
+// Model:                           143
+// In this example, the Model number '143' matches Processor codename: 'SAPPHIRERAPIDS_X' and microarchitecture: 'Golden Cove'.
+func getCPUModel() string {
+	// use lscpu to get CPU Model number
```

Comment:
this function seems to use cpuid not lscpu

Reply:
> this function seems to use cpuid not lscpu

sorry,  here is typo in line 204, should be "lscpu", but as you know, in this PR comments thread, folks gave lots of constructive suggestions to use golang pkg/lib instead of "lscpu", I will figure out a new diff soon.